en/option/partial/group-id.md

```diff
@@ -3,15 +3,59 @@
 
 ## dataGroupId(string)
 
-A groupID common to all data in the series. the groupID will be used to classify the data and determine how merge and split animations are performed in the universal transition animation.
-
-If you are using the [dataset](~dataset) component to represent the data, it is recommended to use `encode.itemGroupID` to specify which dimension is encoded as the groupID.
+A group ID assigned to all data items in the series.
 
+This option has a lower priority than `groupId`, which means when `groupId` is specified for a certain data item the `dataGroupId` will be simply ignored for that data item. For more information, please see `series.data.groupId`.
 
 
 {{ target: partial-data-group-id }}
 
 #${prefix} groupId(string)
 
-The groupID of this data item. groupID will be used to classify the data.
+The group ID of a data item. When `universalTransition` is enabled, the data items from the old option and those from the new one, if sharing a same group ID, will then be matched and applied to a proper animation after `setOption` is called.
+
+If a data item is not specified with a `groupId`, Echarts will try to use `series.dataGroupId` as the group ID for the data item. If `series.dataGroupId` is not specified either, Echarts will fall back to using the data item's ID as its group ID.
+
+If you are using the [dataset](~dataset) component to represent data, you are recommended to use `encode.itemGroupId` to specify the dimension that is to be encoded as the group ID.
+
+
+{{ target: partial-data-child-group-id }}
+
+#${prefix} childGroupId(string)
+
+The group ID of the child data of a data item. This option is introduced to make multiple levels drilldown and aggregation animation possilbe.
+
+~[700x300](${galleryViewPath}doc-example/bar-drilldown&edit=1&reset=1)
```

Comment:
This example will take effect after https://github.com/apache/echarts-examples/pull/55 is finished.

Reply:
It seems the new options don't have a version tip, such as `childGroupId` and `itemChildGroupId`. You can add it like this,

```
{{ use: partial-version(
    version = "5.5.0"
) }}
```

And the example link might need to be updated to the new `bar-multi-drilldown`.

Reply:
Got it, I will create another PR to address this :-)